contracts/ICO_controller.sol

```diff
@@ -236,14 +259,14 @@ contract ICO_controller is Ownable, TransferableInterface {
         }
         // burn after airdrop left tokens
         if (totalAirdropAdrresses != 0) {
-            uint256 airdropToBurn = AIRDROP_SUPPLY.sub(AIRDROP_SUPPLY.div(totalAirdropAdrresses).mul(totalAirdropAdrresses));
+            uint256 airdropToBurn = airdropSupply.sub(airdropSupply.div(totalAirdropAdrresses).mul(totalAirdropAdrresses));
             
             if (airdropToBurn != 0 && controllerCurrentBalance >= airdropToBurn){
                 token.burn(airdropToBurn);
             }
         } else {
-            if (controllerCurrentBalance >= AIRDROP_SUPPLY) {
-                token.burn(AIRDROP_SUPPLY);
+            if (controllerCurrentBalance >= airdropSupply) {
```

Comment:
Not related to this change: I think we have a bug related to `controllerCurrentBalance` - do we need to re-fetch the balance after we do `token.burn(unspendDevReward)` (and before trying to burn airdrop tokens)?